Content.Server/Traits/Assorted/NarcolepsyComponent.cs

```diff
@@ -20,5 +20,21 @@ public sealed partial class NarcolepsyComponent : Component
     [DataField("durationOfIncident", required: true)]
     public Vector2 DurationOfIncident { get; private set; }
 
+    [DataField] // # DeltaV begin Narcolepsy port from EE
```

Comment:
autopausedfield + timeoffsetserializer

Reply:
this requires i turn a float into a timespan and i am lamenting

Reply:
oh it's a fucking float-
why...
this should be stored as a timespan

Reply:
i think it'd be better to get it as a TimeSpan.FromSeconds() inside of PrepareNextIncident() and just refactor everything else to use timespans (or get it from seconds for the ResetNarcolepsy method)

Reply:
fwiw, upstream has NextIncidentTime as a float, and if this is absolutely a requirement ok- but this would be a better PR there then. 

![image](https://github.com/user-attachments/assets/b9dc2e9d-d95d-489e-a544-f3ddf958d015)

Refactoring the entire thing is beyond my ability- so please close this out if so